Reekoh/raygun-exception-handler
# Master

To test:
go to - https://raygun.io/
sign in using these credentials - email : adinglasan@reekoh.com pass : achilles123
run gulp test and check if exception is recorded on the raygun dashboard.
